src/common/constants.ts

```diff
@@ -1,14 +1,23 @@
+import { env } from "node:process";
+
 /**
  *  @internal
  *  @module Constants
  */
 
+const parseInt = (str: string | undefined): number | undefined => {
+    if (str == undefined) return undefined;
```

Comment:
is it intentional we are not using triple equals here? 🤔

Reply:
Oh oops my mistake!! Weird the linter didn't catch it, thanks!